runtime/compiler/z/runtime/PicBuilder.m4

```diff
@@ -1961,9 +1961,28 @@ ZZ 31 bit zOS. See definition of J9TR_CAA_SAVE_OFFSET
 
 })dnl
 
-LOAD_ADDR_FROM_TOC(r14,TR_instanceOf)
+ifdef({J9ZOS390},{dnl
+
+LOAD_ADDR_FROM_TOC(r6,TR_instanceOf)
```

Comment:
Looking at the macro defintion and other uses of LOAD_ADDR_FROM_TOC, it seems like this loads the function descriptor for TR_instanceOf.   Can you confirm we do not need a ` LMG r5,r6,0(r6)` to load the env / function target?   i.e. https://github.com/eclipse-openj9/openj9/pull/14443/files#diff-6870186d0528e479ad4329dd84386da041f665c28e65843be2f05f9406b8e5a1L1842-L1847

```
define(LOAD_ADDR_FROM_TOC,
{dnl
ZZ Load TOC address
   L_GPR $1,J9TR_VMThread_codertTOC(,r13)
ZZ Load callee address
   L_GPR $1,(($2-1)*PTR_SIZE)(,$1)
})dnl
```

Reply:
Helpers entered in [`CodeRuntimeHelperTable`](https://github.com/eclipse-openj9/openj9/blob/a245e34090b3049f0339db03002a7a4a1c3b3460/runtime/compiler/runtime/Runtime.cpp#L966) with the `SET` macro are in the table just as function addresses, not function descriptors, so they don't have the extra level of indirection that makes `LMG r5,r6,0(r6)` necessary. Only helpers entered with the `SET_CONST` macro use function descriptors and need to load the env / function target.